### PR TITLE
Trello button and link changes

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

Remove the icons on the Kubernetes Detail pages and change the "Add a Node Pool" button to primary
Change the button to a link on `My Profile > Password & Authentication > Password Reset`
Change the button style from primary to secondary on `Firewalls > Rules`

### CODE REVIEW

File: packages/manager/src/features/Profile/AuthenticationSettings/ResetPassword.tsx [hunk 1]
```diff
@@ -37,19 +40,17 @@ export const ResetPassword: React.FC<CombinedProps> = props => {
 
   return (
     <Paper className={`${classes.root} ${disabled ? classes.disabled : ''}`}>
-      <Typography variant="h3">Password Reset</Typography>
-      <Typography variant="body2" className={classes.copy}>
+      <Typography variant="h2">Password Reset</Typography>
```
Comment (tiffwong) [inline packages/manager/src/features/Profile/AuthenticationSettings/ResetPassword.tsx:43]: Adjusted the variant here to match the styles used in the TFA section below

File: packages/manager/src/features/Profile/AuthenticationSettings/ResetPassword.tsx [hunk 2]
```diff
@@ -37,19 +40,17 @@ export const ResetPassword: React.FC<CombinedProps> = props => {
...
+      <Typography variant="body1" className={classes.copy}>
```
Comment (tiffwong) [inline packages/manager/src/features/Profile/AuthenticationSettings/ResetPassword.tsx:44]: Next Level Apparel

Comment (tiffwong) [inline packages/manager/src/features/Profile/AuthenticationSettings/ResetPassword.tsx:44]: Adjusted the variants here to match the ones used in the TFA section below

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodePoolsDisplay.tsx [hunk 3]
```diff
@@ -243,19 +237,18 @@ export const NodePoolsDisplay: React.FC<Props> = props => {
...
-          <div
+          <Button
+            buttonType="primary"
             className={classnames({
-              [classes.addNodePoolLink]: true,
+              [classes.button]: true,
               [classes.cmrSpacing]: flags.cmr
             })}
+            onClick={() => {
+              handleOpenAddDrawer();
```
Comment (johnwcallahan) [inline packages/manager/src/features/Kubernetes/KubernetesClusterDetail/NodePoolsDisplay/NodePoolsDisplay.tsx:247]: No need to create a new function here; you can do: `onClick={handleOpenAddDrawer}`.

**Review by johnwcallahan — CHANGES_REQUESTED** [review]
This is also an issue in production, but we might as well fix it here:
Screen Shot 2020-12-18 at 9 02 47 AM

**Review by DevDW — APPROVED** [review]
I didn't see any additional issues beyond what @johnwcallahan pointed out. Approved pending on those fixes

**Review by johnwcallahan — APPROVED** [review]

**Pull request merged**